bitstreams.md

```diff
@@ -150,10 +150,40 @@ Return codes:
 * 422 Unprocessable Entity - if the provided uuid cannot be resolved to an item regardless to the item status
 
 ## DELETE Method
+
+### Single Bitstream Delete
 Delete a bitstream. Works for normal bitstreams in an Item (bundle), and a community or collection logo
 
 * 204 No content - if the operation succeed
 * 401 Unauthorized - if you are not authenticated
 * 403 Forbidden - if you are not loggedin with sufficient permissions
 * 404 Not found - if the bitstream doesn't exist (or was already deleted)
 * 422 Unprocessable Entity - if the bitstream is a community or collection logo
+
+### Multiple Bitstreams Delete
```

Comment:
I feel this is on the wrong REST API endpoint.  The idea is fine with me, but I think we should move this to `/api/core/bundles/<:uuid>` so that it aligns with the existing PATCH operations on that endpoint.  See existing bundle PATCH docs (for reordering bitstreams) at https://github.com/DSpace/RestContract/blob/main/bundles.md#ordering-bitstreams

So, I'd propose we enhance the existing PATCH options on bundle to allow for multiple Bitstream removal similar to this:
```
curl -i -X PATCH 'https://api7.dspace.org/server/api/core/bitstreams -H 'Authorization: Bearer eyJhbGciO…' -H "content-type: application/json"
--data '[ { "op": "remove", "path": "/_links/bitstreams/0"}, { "op": "remove", "path": "/_links/bitstreams/1"}]'
```

I realize the PATCH body syntax there isn't great though because it reference Bitstreams by their index instead of by their UUID.  Not sure if there's a way around that, but I wanted to achieve better alignment with the "move" action already on that endpoint.

Reply:
Resolving this feedback... see my [follow-up comment below](https://github.com/DSpace/RestContract/pull/203#issuecomment-1517997141).